notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1139 @@
+{
```

Comment:
<p>This looks much clearer! Great job refactoring it. Maybe add a comment on that <code>projection_shape. is</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Thank you!</p><p>I rewrote the markdown above to include <code>projection_shape</code>.</p>